styles.css

```diff
@@ -103,6 +103,19 @@ header.toolbar img {
   width: 1.875rem;
 }
 
+small {
+  display: none;
+}
+
+small.form-message {
+  display: flex;
+  color: #f00;
+  margin-bottom: 1rem;
+  justify-content: center;
+  background-color: #ebe7e0;
```

Comment:
- [ ] [OPTIONAL] Good job implementing the changes requested by the previous reviewer :clap:  I would suggest to drop the background color.